crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -375,43 +370,16 @@ async fn execute_transactions(
 ) -> SuiResult<Vec<SignedTransactionEffects>> {
     let tx_digests: Vec<TransactionDigest> =
         execution_digests.iter().map(|tx| tx.transaction).collect();
-    let mut txns: Vec<VerifiedCertificate> = authority_state
+    let txns: Vec<VerifiedCertificate> = authority_state
         .database
-        .epoch_store()
-        .multi_get_pending_certificate(&tx_digests)?
+        .perpetual_tables
+        .synced_transactions
```

Comment:
How is the `synced_transactions` table cleaned up?

Reply:
Yeah gotta figure this out still...